plugins/nf-amazon/src/main/nextflow/cloud/aws/batch/AwsBatchHelper.groovy

```diff
@@ -181,7 +181,9 @@ class AwsBatchHelper {
         final response = batchClient.describeJobs(request)
         if( response.jobs ) {
             final detail = response.jobs[0]
-            return detail.container.logStreamName
+            return detail.ecsProperties?.taskProperties[0]?.containers[0]?.logStreamName
```

Comment:
Not sure if with an empty list `[0]` returns `IndexOutOfBoundException`, may be it's better to wrap this (and other similar) in helper method and use if try/catch(Exception) returning a `null` if something is going wrong

Reply:
Just checked and it seems `emptyList[0]` returns null so I think that's fine. Unfortunately `obj.nullList[0]` throws a NPE.

This is a bit awkward, but would work:
```groovy
detail.ecsProperties?.taskProperties?[0]?.containers?[0]?.logStreamName
```

Reply:
Too ugly, make sense to have little helper method